optaplanner-wb-screens/optaplanner-wb-domain-editor/optaplanner-wb-domain-editor-client/pom.xml

```diff
@@ -38,6 +38,10 @@
       <artifactId>gwtbootstrap3</artifactId>
     </dependency>
     <dependency>
+      <groupId>org.gwtbootstrap3</groupId>
```

Comment:
If you're adding use of GWT-BS3's ```Select``` you also need include an additional ```gwt.xml``` module.

i.e. ```<inherits name="org.gwtbootstrap3.extras.select.Select"/>```

See https://gwtbootstrap3.github.io/gwtbootstrap3-demo/#select

I couldn't see that in this commit.